### PR TITLE
allow session expiry to be configurable

### PR DESCRIPTION
**Pull request by dylmax12**
Allow the session expiry to be configurable via config file

### CODE REVIEW

File: src/conf.c [hunk 1]
```diff
@@ -2188,6 +2202,7 @@ static int config__read_file_core(struct mosquitto__config *config, bool reload,
 #endif
 				}else{
 					log__printf(NULL, MOSQ_LOG_ERR, "Error: Unknown configuration variable \"%s\".", token);
+                    log__printf(NULL, MOSQ_LOG_ERR, "Error: hasdfgasdfasfasdfasdfasdf \"%s\".", token);
```
Comment (calliday) [inline src/conf.c:2205]: I think you forgot to remove a debug log! :)

File: src/bridge.c [hunk 2]
```diff
@@ -348,6 +348,11 @@ int bridge__connect(struct mosquitto *context)
 	uint8_t notification_payload;
 	uint8_t qos;
 
+    //mosquitto_property receive_maximum
```
Comment (esoutham-lvt) [inline src/bridge.c:351]: Are these additional properties that you intend to also make configurable in the future?

File: src/conf.c (hunk 1 above)
Comment (esoutham-lvt) [inline src/conf.c:2205]: Did you forget to delete this additional comment?

**Review by esoutham-lvt — APPROVED** [review]
Did you forget to delete this additional comment?

File: src/bridge.c (hunk 2 above)
Comment (dylmax12) [reply src/bridge.c]: Right now, this is the only one we anticipate, but yes in theory more could be added in the future. We anticipate as mosquitto builds better support for MQTT 5 features that we wont have to add these patches in ourselves but rather they would be a part of the official moquitto application.

**Review by calliday — APPROVED** [review]

**Review by smartsystemdesign — APPROVED** [review]

**Review by calliday — APPROVED** [review]

**Pull request merged**